unity/CAVE/Assets/Editor/CLI.cs

```diff
@@ -69,156 +65,152 @@ public static void Main()
         }
 
         // Get command line arguments from Python
-        private static string GetXmlPathArg()
+        private static void GetXmlPathArg()
         {
             try
             {
                 string[] args = Environment.GetCommandLineArgs();
                 for (int i = 0; i < args.Length; i++)
                 {
-                    if (args[i] == "--xmlPath") { return args[++i]; }
+                    if (args[i] == "--projectPath") { ProjectPath = args[++i]; }
```

Comment:
How do the arguments interact with the scripts being run directly through unity?

Reply:
It's a function of `System.Environment` not Unity unfortunately. They're only passed when I call the unity editor from the command line so I have no way seeing/using them from the GUI

Reply:
I thought that might be the case. Should we consider adding them to the GUI, and having the GUI values overwrite the environment values? Or maybe the GUI values can be prepopulated with the environment variables?

Reply:
Yes, right now the GUI and environment are both interacting with the `CLI.ProjectPath` string. The GUI checks to see if that value is null before updating it to what I've hard coded. If someone where to run the script first the GUI would see the prepopulated value